src/MeoAsstGui/Resources/Localizations/ja-jp.xaml

```diff
@@ -179,28 +179,28 @@
     <system:String x:Key="Hibernate">休止状態にする*</system:String>
     <system:String x:Key="Shutdown">シャットダウン*</system:String>
 
-    <system:String x:Key="LinkStart">Link Start!</system:String>
+    <system:String x:Key="LinkStart">Link Start!</system:String><!--分かりやすく動作開始でも良い気はする-->
     <system:String x:Key="Stop">止める</system:String>
 
     <!--  戦闘設定  -->
-    <system:String x:Key="UseSanityPotion">回復剤使う</system:String>
-    <system:String x:Key="UseOriginitePrime">源石使う*</system:String>
-    <system:String x:Key="PerformBattles">回数限定*</system:String>
-    <system:String x:Key="AssignedMaterial">素材限定</system:String>
+    <system:String x:Key="UseSanityPotion">理性回復剤の使用数</system:String><!--UIの表示バランス大丈夫？-->
+    <system:String x:Key="UseOriginitePrime">純正源石の使用数*</system:String><!--UIの表示バランス大丈夫？-->
+    <system:String x:Key="PerformBattles">周回数指定*</system:String>
+    <system:String x:Key="AssignedMaterial">素材を限定</system:String>
     <system:String x:Key="Quantity">ドロップ数</system:String>
     <system:String x:Key="StageSelect">ステージ</system:String>
-    <system:String x:Key="StageSelect2">代わり</system:String>
-    <system:String x:Key="Amend">改定</system:String>
+    <system:String x:Key="StageSelect2">実行できない際の代替</system:String>
```

Comment:
If you use only "代替" in Japanese, it is not clear what is being substituted, so it would be better to annotate with a tip.

Reply:
If you use only "代替" in Japanese, it is not clear what is being substituted, so it would be better to annotate with a tip.

Reply:
what about 「候補」

Reply:
Same.
I think that if you don't annotate with hints, etc., the operation will not be understood in Japanese.

A candidate would be smarter to add a display of annotations of the contents of the action to the empty space at the bottom of the UI.

Or add an addendum to the manual, "Please read the manual." to guide the user.

Reply:
Yes.
But in Chinese client we use 备选(Candidate), which is similar to the expression in Japanese client.
Since users will see this word only if they have checked this option in settings by their own, we don't need to worry about their misunderstanding.

Reply:
what about 「代替ステージ」

Reply:
If the above is correct, there should be no problem.

Reply:
～